rust/bridge/ffi/cbindgen.toml

```diff
@@ -17,8 +17,10 @@ style = "type"
 prefix_with_name = true
 
 [export]
-include = ["SignalErrorCode", "FfiDirection", "FfiCiphertextMessageType", "FfiContentHint"]
-item_types = ["enums", "functions", "opaque", "structs", "typedefs"]
+include = ["SignalErrorCode", "FfiDirection", "FfiCiphertextMessageType", "FfiContentHint", "RandomnessBytes"]
+exclude = ["TAG_SIZE", "NONCE_SIZE"]
+item_types = ["enums", "functions", "opaque", "structs", "typedefs", "constants"]
+# FIXME: this doesn't work well with constants in SCREAMING_SNAKE_CASE
```

Comment:
What is this referring to?

Reply:
The following line, `prefix = "Signal"`. Constants end up as `SignalFOO_BAR` rather than `SIGNAL_FOO_BAR`. But since the C API isn't an end product I decided it wasn't worth spending time on.